dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java

```diff
@@ -376,7 +429,6 @@ public void testFolderWithFileMove()
     Assert.assertTrue(b.isDirectory());
     Assert.assertTrue(dstD.exists());
     Assert.assertTrue(dstD.isDirectory());
-    assertFalse(a.exists());
```

Comment:
I'm a bit confused, if a still exists meaning we fail to delete a, then why we do not get any exceptions?

Reply:
Looks like there is a bug here. I don't why there isn't any exceptions. It shows that it succeeds to delete, but the directory is still there.